src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken');
+
+exports.authenticateToken = (req, res, next) => {
+  const authHeader = req.headers.authorization;
+  const token = authHeader && authHeader.split(' ')[1];
```

Comment:
This will probably throw an error or lead to unwanted behavior if `[1]` is not defined